pkg/utils/common.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"crypto/x509"
+
+	"github.com/ansible/receptor/pkg/logger"
+)
+
+func ParseReceptorNamesFromCert(cert *x509.Certificate, expectedHostname string) (bool, []string, error) {
+	var receptorNames []string
+	receptorNames, err := ReceptorNames(cert.Extensions)
```

Comment:
if ReceptorNames is no longer being called outside of the utils package, then we can "unexport" it -- make it lower case